modules/apps/commerce/commerce-price-list-service/src/main/java/com/liferay/commerce/price/list/service/impl/CommerceTierPriceEntryLocalServiceImpl.java

```diff
@@ -211,8 +211,9 @@ public CommerceTierPriceEntry addCommerceTierPriceEntry(
 
 		// Commerce price entry
 
-		_commercePriceEntryLocalService.setHasTierPrice(
-			commercePriceEntryId, true, bulkPricing);
+		CommercePriceEntryUtil.setHasTierPrice(
```

Comment:
@dantewang This change is going to skip the reindex in _commercePriceEntryLocalService#setHasTierPrice. I did not see the circular dependencies here, could you please add the details to the commit message?

Reply:
@tinatian As far as I remember, a delete logic in `CommercePriceEntryLocalService` uses the delete method on `CommerceTierPriceEntryLocalService`. I'll double check.